lib/flatpak-installation.h

```diff
@@ -76,6 +76,17 @@ typedef enum {
   FLATPAK_INSTALL_FLAGS_NO_PULL          = (1 << 3),
 } FlatpakInstallFlags;
 
+/**
+ * FlatpakUninstallFlags:
+ * @FLATPAK_UNINSTALL_FLAGS_NONE: Default
```

Comment:
Missing doc string for the NO_PRUNE flag

Reply:
Ok